### PR TITLE
[cppdocs] Note on Tensor Creation

### PR DESCRIPTION
**Pull request by goldsborough**

### CODE REVIEW

File: docs/cpp/source/tensor_creation.rst [hunk 1]
```diff
@@ -0,0 +1,279 @@
...
+
+.. code-block:: cpp
+
+  torch::ones(10, torch::TensorOptions().dtype(torch::kFloat32))
+
+is equivalent to
+
+.. code-block:: cpp
+
+  torch::ones(10, torch::dtype(torch::kFloat32))
```
Comment (ebetica) [inline docs/cpp/source/tensor_creation.rst]: Would it be useful to mention that you can actually use the constructor of TensorOptions to even further shorten this to ```torch::ones(10, torch::kFloat32)``` ?

**Review by ebetica — APPROVED** [review]
This is awesome! Thanks!

File: docs/cpp/source/tensor_creation.rst (hunk 1 above)
Comment (goldsborough) [reply docs/cpp/source/tensor_creation.rst]: Oh yes absolutely, thanks

**Review by ezyang — APPROVED** [review]
